src/VideoManager/SubtitleWriter.cc

```diff
@@ -22,6 +22,15 @@
 #include <QDate>
 
 QGC_LOGGING_CATEGORY(SubtitleWriterLog, "SubtitleWriterLog")
+const QString SubtitleWriter::vehicleFactGroupName = QStringLiteral("Vehicle");
+
+const QStringList SubtitleWriter::settingsSources = QStringList{
+    QStringLiteral("TelemetryBarUserSettingsWIP02-0"),
```

Comment:
Hi @DonLakeFlyer

I noticed this needed some more changes to handle the new instruments widget, but I have some questions:

Are [these](https://github.com/mavlink/qgroundcontrol/blob/master/src/QmlControls/HorizontalFactValueGrid.cc#L17) still WIP?

I can't figure out where the `-0` and `-2` came from... Culd you point me in the right direction?
Thanks!

Reply:
The WIP should be removed from these settings names, since they are no longer WIP. That said you shouldn't be reading these settings directly. With the current code what you want to do can't really be done. It requires some refactoring of HorizontalFactValueGrid and FactValueGrid. Is this a new feature or something that was working before you are now trying to get working again?

Reply:
I was trying to fix #8766, but I realized that the new implementation broke it, so I'm trying to fix that too.

The behavior in 4.0 is that it reads all the items from the values widgets and uses them for the [video overlay](https://docs.qgroundcontrol.com/master/en/FlyView/VideoOverlay.html).

I'll take a good look at HorizontalFactValueGrid and FactValueGrid tomorrow.

Reply:
> The behavior in 4.0 is that it reads all the items from the values widgets and uses them for the [video overlay](https://docs.qgroundcontrol.com/master/en/FlyView/VideoOverlay.html).

Ah, crap. Ok, let me clean up a couple things and then you should be able to create a standalone HorizontalFactValueGrid and read from it.